release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -502,6 +502,10 @@ The vSphere Problem Detector Operator periodically checks functionality of {prod
 
 The Local Storage Operator now includes a must-gather image, allowing you to collect custom resources specific to this Operator for diagnostic purposes. See link:https://bugzilla.redhat.com/show_bug.cgi?id=1756096[*BZ#1756096*] for more information.
 
+[id="ocp-4-7-aws-efs-tp_removed-1st"]
+==== External provisioner for AWS EFS (Technology Preview) feature has been removed
+The Amazon Web Services (AWS) Elastic File System (EFS) technology preview feature has been removed and is no longer supported.
+
```

Comment:
We have claimed that EFS was removed in the 4.5 release note, in 4.7, we marked it in Table 1. Deprecated and removed features tracker as REM should be enough?
In this chapter, looks like we only doc the new changes in this version.

Reply:
I talked to Jan about this and we agreed that since we are retroactively removing this feature from all releases, rather than actually removing it starting with 4.5, that we should include the removal notice in all versions of the rel notes. Make sense?